Source/IDE Wizard/ServerMainFormSRC.txt

```diff
@@ -7,15 +7,18 @@ unit ServerMainForm;
 
 interface
 
-uses Classes, SysUtils, Forms, ActnList, ComCtrls, StdCtrls, Controls, ExtCtrls
-  , Diagnostics
-
-  , MARS.Core.Engine
-  , MARS.http.Server.Indy
-
-  , MARS.Core.Application
-  , System.Actions
-  ;
+uses
+  System.Classes, System.SysUtils, System.Diagnostics, System.Actions,

```

Comment:
I don't know what the minimum Delphi version for MARS is, but the namespaces are relatively new...

Reply:
Currently most features should be available with Delphi XE (server side).
The client library has Async support only from XE7 and later.
I am currently thinking about moving the MARS minimum Delphi version somewhere near Delphi 10 (Seattle). This would greatly eliminate IFDEFs across the project but I don't know how this would impact the user base...